packages/grafana-ui/src/components/Combobox/getComboboxStyles.ts

```diff
@@ -53,6 +57,9 @@ export const getComboboxStyles = (theme: GrafanaTheme2) => {
       label: 'grafana-select-option-label',
       textOverflow: 'ellipsis',
       overflow: 'hidden',
+      fontSize: MENU_ITEM_FONT_SIZE,
+      fontWeight: 500,
+      letterSpacing: 0, // pr todo: text in grafana has a slightly different letter spacing, which causes measureText() to be ~5% off
```

Comment:
The complication of `measureText()` is that we need to make sure the text is styled _exactly_ the same across that canvas and in the page.